cli/upload/upload.go

```diff
@@ -109,6 +115,28 @@ func runUploadCommand(command *cobra.Command, args []string) {
 	})
 	if err != nil {
 		feedback.Errorf(tr("Error during Upload: %v"), err)
+
+		// Check the error type to give the user better feedback on how
+		// to resolve it
+		var platformErr *arduino.PlatformNotFoundError
+		if errors.As(err, &platformErr) {
+			split := strings.Split(platformErr.Platform, ":")
+			if len(split) < 2 {
+				panic(tr("Platform ID is not correct"))
+			}
+
+			pm := commands.GetPackageManager(instance.GetId())
+			platform := pm.FindPlatform(&packagemanager.PlatformReference{
+				Package:              split[0],
+				PlatformArchitecture: split[1],
+			})
+
+			if platform != nil {
+				feedback.Errorf(tr("Try running `%s core install %s`", globals.VersionInfo.Application, platformErr.Platform))
```

Comment:
```suggestion
				feedback.Errorf(tr("Try running %s", fmt.Sprintf("`%s core install %s`", globals.VersionInfo.Application, platformErr.Platform)))
```
See explanation above.